src/gsi/deter_sfc_mod.f90

```diff
@@ -1331,7 +1331,7 @@ subroutine deter_sfc_amsre_low(dlat_earth,dlon_earth,isflg,sfcpct)
    end subroutine deter_sfc_amsre_low
 
 
-subroutine deter_sfc_gmi(dlat_earth,dlon_earth,isflg,sfcpct)
```

Comment:
`sfcpct` has been removed as a subroutine argument.  However, comments on lines 1357 to 1361 still document `sfcpct`.  Remove the comments since they are no longer relevant.

Reply:
Removed. Thanks.